### DISCUSSION TITLE
run k3s agent on risc-v

### DISCUSSION THREAD

**Discussion by lxzan** [k3s-io/k3s#10548] [Q&A]
How are mirrored-library-traefik and klipper-lb packaged, and where can I see the build scripts ? I would like to port them to RISC-V.

**Comment by dereknola** [comment]
Traefik is mirror via the https://github.com/rancher/image-mirror/blob/master/images-list#L564, (its not rebuild its just hosted as a mirror). If you want to port them to RISC-V you need to go add support directly into https://github.com/traefik/traefik. Klipper-lb repo is https://github.com/k3s-io/klipper-lb. Please use the existing issue https://github.com/k3s-io/k3s/issues/7151
